src/Internal/Material/PasswordInput.elm

```diff
@@ -0,0 +1,92 @@
+module Internal.Material.PasswordInput exposing (passwordInput, passwordInputBase, searchInput)
+
+import Element
+import Element.Border as Border
+import Internal.Material.Chip as Chip
+import Internal.Material.Palette exposing (Palette)
+import Internal.PasswordInput exposing (PasswordInputStyle)
+import Widget.Customize as Customize
+import Widget.Material.Color as MaterialColor
+
+
+passwordInput : Palette -> PasswordInputStyle msg
+passwordInput palette =
+    { elementRow =
+        (palette.surface
+            |> MaterialColor.textAndBackground
+        )
+            ++ [ Element.spacing 8
+               , Element.paddingXY 8 0
+               , Border.width 1
+               , Border.rounded 4
+               , palette.on.surface
+                    |> MaterialColor.scaleOpacity 0.14
+                    |> MaterialColor.fromColor
+                    |> Border.color
+               , Element.focused
+                    [ Border.shadow <| MaterialColor.shadow 4
+                    , palette.primary
+                        |> MaterialColor.fromColor
+                        |> Border.color
+                    ]
+               , Element.mouseOver [ Border.shadow <| MaterialColor.shadow 2 ]
+               ]
+    , content =
+        { password =
+            { elementPasswordInput =
+                (palette.surface
+                    |> MaterialColor.textAndBackground
+                )
+                    ++ [ Border.width 0
+                       , Element.mouseOver []
+                       , Element.focused []
+                       , Element.centerY
+                       ]
+            }
+        }
+    }
+
+
+searchInput : Palette -> PasswordInputStyle msg
```

Comment:
Why is there a `searchInput` style implemented? Can this be removed or is it relevant to the password input?

Reply:
my bad, I did not clean up as well as I thought